examples/demos/example27-cell-actions.tsx

```diff
@@ -147,38 +146,19 @@ const columns = [
   }
 ];
 
-export default class extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = { rows: createRows(2000) };
-  }
-
-  getRows = () => {
-    return Selectors.getRows(this.state);
-  };
-
-  getRowAt = (index) => {
-    const rows = this.getRows();
-    return rows[index];
-  };
+export default function() {
+  const [rows] = useState(() => createRows(2000));
 
-  getSize = () => {
-    return this.getRows().length;
-  };
-
-  render() {
-    return (
-      <Wrapper title="Cell Actions Example">
-        <DataGrid
-          ref={node => this.grid = node}
-          enableCellSelect
-          columns={columns}
-          rowGetter={this.getRowAt}
-          rowsCount={this.getSize()}
-          rowHeight={55}
-          minHeight={600}
-        />
-      </Wrapper>
-    );
-  }
+  return (
+    <Wrapper title="Cell Actions Example">
+      <DataGrid<Row, 'id'>
+        enableCellSelect
+        columns={columns}
+        rowGetter={i => rows[i]}
```

Comment:
It doesn't matter too much for demos, but we should avoid passing new functions on each render to DataGrid.

Reply:
I am thinking of using `eventBus` more aggressively and avoid passing callbacks down. I have a few ideas that we can discuss next week